setup.cfg

```diff
@@ -1,13 +1,13 @@
 [metadata]
-name = tiledbimg
+name = tiledb-bioimg
 description = Package supports all bio-imaging functionality provided by TileDB
 author = TileDB, Inc.
 author_email = help@tiledb.io
 long_description = file: README.md
 long_description_content_type = text/markdown
 license = MIT
 license_file = LICENSE
-keywords = tiledb, tiledbimg , bioimaging
+keywords = tiledb, tiledb-bioimg , bioimaging
```

Comment:
`tiledb-bioimg` doesn't make much sense as a keyword

Reply:
Borrowed it from `ML`. Can remove